Xamarin.Forms.Core/Button.cs

```diff
@@ -49,9 +49,19 @@ public class Button : View, IFontElement, ITextElement, IBorderElement, IButtonC
 
 		public static readonly BindableProperty ImageProperty = ImageElement.FileImageProperty;
 
-
 		public static readonly BindableProperty PaddingProperty = PaddingElement.PaddingProperty;
 
+		public static readonly BindableProperty LineBreakModeProperty = BindableProperty.Create(nameof(LineBreakMode), typeof(LineBreakMode), typeof(Button), LineBreakMode.WordWrap,
```

Comment:
I see you’ve added the `LineBreakMode` to address the feedback, but it doesn’t seem to be implemented? Just like the `MaxLines` this will also need some work in the renderers to make `MaxLines` and `LineBreakMode` work together or make `LineBreakMode` work at all.